ape_alchemy/provider.py

```diff
@@ -203,6 +203,11 @@ def make_request(self, rpc: str, parameters: Optional[Iterable] = None) -> Any:
             )
             raise cls(message) from err
 
+        if isinstance(result, dict) and (res := result.get("result")):
```

Comment:
this is what `ape-ethereum` expects: https://github.com/ApeWorX/ape/blob/main/src/ape_ethereum/provider.py#L1127-L1130